Packs/CarbonBlackDefense/Integrations/CarbonBlackLiveResponseCloud/CarbonBlackLiveResponseCloud.py

```diff
@@ -0,0 +1,363 @@
+    return f'File: {file_id} is successfully put to the remote destination {destination_path}'
+
+
+# Using Py API
+def get_file_command(api_client: CBCloudAPI, device_id: str, source_path: str, timeout: Union[int, str] = None,
+                     delay: Union[float, str] = None):
+    session = api_client.select(platform.Device, device_id).lr_session()
+
+    if delay:
+        delay = float(delay)
+    file_data = session.get_file(file_name=source_path, timeout=arg_to_number(timeout), delay=delay)
+    file_name = ntpath.split(source_path)[1]
+    return fileResult(file_name, file_data)
+
+
+# Using Py API
+def delete_file_command(api_client: CBCloudAPI, device_id: str, source_path: str):
+    session = api_client.select(platform.Device, device_id).lr_session()
+    session.delete_file(filename=source_path)
+    return f'The file: {source_path} was deleted successfully.'
+
+
+# Using Py API
+def list_directory_command(api_client: CBCloudAPI, device_id: str, directory_path: str, limit: Union[int, str]):
+    """
+       Get list of directory entries in the remote device
+
+       :param api_client: The API client
+
+       :param device_id: The device id
+
+       :param directory_path: Directory to list. This parameter should end with the path separator
+
+       :param limit: Limit the result entries count to be the given limit
+
+       :return: CommandResult represent the API command result
+       :rtype: ``CommandResults``
+    """
+    session = api_client.select(platform.Device, device_id).lr_session()
+    items = [item for item in session.list_directory(directory_path) if item['filename'] not in IGNORED_FILES_IN_DIR]
+    items, partial_res_msg = get_limited_results(original_results=items, limit=limit)
+
+    directories_readable = []
+    context_entry_items = []
+    headers = ['name', 'type', 'date_modified', 'size']
+    for item in items:
+        context_entry_items.append(item)
+        directories_readable.append({
+            'name': item['filename'],
+            'type': 'Directory' if item['attributes'] and 'DIRECTORY' in item['attributes'] else 'File',
+            'date_modified': item['last_write_time'],
+            'size': item['size']
+        })
+
+    context_entry = dict(content=context_entry_items, device_id=device_id, directory_path=directory_path)
+
+    readable_output = tableToMarkdown(f'Directory of {directory_path}{partial_res_msg}',
+                                      t=directories_readable,
+                                      headers=headers,
+                                      headerTransform=string_to_table_header,
+                                      removeNull=True)
+
+    return CommandResults(
+        outputs_prefix='CarbonBlackDefenseLR.Directory',
+        outputs_key_field=['device_id', 'directory_path'],
+        outputs=context_entry,
+        readable_output=readable_output,
+        raw_response=items
```

Comment:
```suggestion
        raw_response=items,
```

Update in all relevant places.